src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#home" class="icon">
+        <img
+        src="images/logo.png"
+        alt="logoMoyo"
+        >
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__container">
```

Comment:
```suggestion
        <ul class="nav__list">
```
ul - unordered list 
let's use semantic names